build.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
```

Comment:
No need to change it. If you say it's common in F# OSS projects, that's another good reason. I any case, I was suggesting that we replaced the .cmd with a .ps1 file, not to keep both.

But just to reiterate, let's keep the one you've added here.